firmware/configuration/t167_default/Makefile.inc

```diff
@@ -9,6 +9,7 @@
 
 F_CPU = 16000000
 DEVICE = attiny167
+AVR_ARCHITECTURE_PARAMETER = --binary-architecture avr:35
```

Comment:
What is this for? Is this needed for attiny167? Then it would make sense to add separately.

Reply:
It is used in the makefile, otherwise I get an error, and this was the quick fix for it.

Reply:
Weird I had no issue when building everything. Which version of avr-gcc are you using?

mine
>gcc version 5.4.0 (AVR_8_bit_GNU_Toolchain_3.6.1_1750)